src/questions/intro-to-lending-borrowing.yaml

```diff
@@ -1,19 +1,404 @@
+      key: C
+    - content: Banking
+      key: D
+
+- uuid: dcfa8605-e19c-41e3-908f-8ab9087d4182
+  type: SingleChoice
+  content: How DeFi's are obtaining real world data since blockchains are isolated in nature?
+  hint: Oracle
+  explanation: Oracles are nodes which provides real world data to the blockchains.
   answerKeys:
     - B
   subTopics:
     - why-do-we-lending-borrowing
-  difficultyLevel: "Low"
+  difficultyLevel: Medium
+  choices:
+    - content: Blockchains are get real world data from internet
+      key: A
+    - content: Through oracles
+      key: B
+    - content: They cant get real world data
+      key: C
+    - content: None of the above
+      key: D
+- uuid: dc3b6825-ee0e-4b79-8116-3063e1386006
+  type: MultipleChoice
+  content: Which of the following are correct about traditional finance?
+  hint: NoHint
+  explanation: TradFi are centralized,simple and less riskier when compared to others. Traditional Finances takes most of the interest from borrowers.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - lending-borrowing-tradFi
+  difficultyLevel: Low
+  choices:
+    - content: Traditional finances are centralized.
+      key: A
+    - content: Traditional Finances takes most of the interest from borrowers.
+      key: B
+    - content: Traditional Finances will provide loans without any agreements and paperworks.
+      key: C
+    - content: Traditional Finances are simple and less riskier.
+      key: D
+- uuid: cc996d05-b62d-43b0-8eaf-a23d7d9d873d
+  type: MultipleChoice
+  content: What are the challenges to privacy in TradFi?
+  hint: NoHint
+  explanation: Users are required to complete their KYC and transactions are monitored in TradFi.
+  answerKeys:
+    - B
+    - C
+  subTopics:
+    - lending-borrowing-tradFi
+  difficultyLevel: Medium
+  choices:
+    - content: Banks check and collect the data from all the legal documents of an user for opening account.
+      key: A
+    - content: Users are required to complete their KYC.
+      key: B
+    - content: Transactions are monitored in TradFi.
+      key: C
+    - content: No personal data is required for lending and borrowing in DeFi.
+      key: D
+
+- uuid: 46633903-58f6-411b-84b9-6cba1a9f8009
+  type: MultipleChoice
+  content: Pick the correct statement(s).
+  hint: NoHint
+  explanation: Traditional finances allows censorship and they have numerous blind spots. They are simple and less risky.
+  answerKeys:
+    - A
+    - B
+    - C
+  subTopics:
+    - lending-borrowing-tradFi
+  difficultyLevel: Medium
+  choices:
+    - content: Traditional Finances allow censorship.
+      key: A
+    - content: Traditional Finances are not transparent.
+      key: B
+    - content: Traditional Finances are simple and less risky when compared to others.
+      key: C
+    - content: None of these
+      key: D
+- uuid: a9513711-0f19-40e8-a9fa-c9c60bb8db70
+  type: SingleChoice
+  content: How users assets are secured by DeFi?
```

Comment:
```suggestion
  content: How user's assets are secured by DeFi?
```

Reply:
Not sure if this is a right question.  We can either as "In DeFi, when lending, who maintains the possession of the user's assets"